test/sass/typograpy.spec.scss

```diff
@@ -191,12 +188,14 @@ $primary-normal: rgb(2, 136, 209) !default;
 
     @include assert() {
 
+      $acolor: rgb(2, 119, 189);
```

Comment:
Prefer to make this obviously local by prefixing with `_`, as mentioned. So `$_acolor`. Do that for all occurrences and I think this will be done.